manifests/params.pp

```diff
@@ -152,6 +152,27 @@
       }
     } # Windows
 
+    'FreeBSD': {
+      $conf_dir  = '/usr/local/etc/icinga2'
+      $log_dir   = '/var/log/icinga2'
+      $run_dir   = '/var/run/icinga2'
+      $spool_dir = '/var/spool/icinga2'
+      $cache_dir = '/var/cache/icinga2'
+      $pki_dir   = "${conf_dir}/pki"
+      $ca_dir    = '/var/icinga2/ca'
+      $user      = 'icinga'
+      $group     = 'icinga'
+      $lib_dir   = '/usr/local/lib/icinga2'
+      $constants = {
+        'PluginDir'          => "${lib_dir}/nagios/plugins",
+        'PluginContribDir'   => "${lib_dir}/nagios/plugins",
```

Comment:
I meant both PluginDir statements, from what I understand of ports (not a FreeBSD user) the location would be:

    /usr/local/libexec/nagios/....

https://svnweb.freebsd.org/ports/head/net-mgmt/monitoring-plugins/Makefile?view=markup&pathrev=431254#l58

We should validate the paths of `net-mgmt/monitoring-plugins`, maybe also test the module on FreeBSD...

`PluginContribDir` is meant for other plugins, that are not included in monitoring-plugins.

But by default the path is set to the same: https://github.com/Icinga/icinga2/blob/8fd454fbb1457a34031a53baa16d2bacc77bba1b/etc/icinga2/constants.conf.cmake#L17